src/commcare_cloud/ansible/roles/supervisor/tasks/main.yml

```diff
@@ -4,8 +4,19 @@
   apt: name=supervisor state=absent
   become: yes
 
+- name: Ensure pip is available
+  apt:
+    name: python3-pip
+    state: present
+  become: yes
+
```

Comment:
```suggestion
```
I think we should put `python3-pip` in the task `Install python packages` or `Install common packages` in `src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml` instead.

Reply:
Thanks. https://github.com/dimagi/commcare-cloud/pull/5211/commits/f47650e40a67517def5b6fe52cfd661f4161d7bd

I still find it difficult to track imports vs dependencies, but your comments have been useful pointers on how to do that.